external/webclient/webclient.c

```diff
@@ -1161,13 +1161,16 @@ static pthread_addr_t wget_base(void *arg)
 		goto errout_before_init;
 	}
 
+	ndbg("Length Info: sndlen : %d buf_len : %d param->buglen : %d\n", strlen(ws.buffer), strlen(param->buffer), param->buflen);
```

Comment:
If content is an information, you should use nvdbg, not ndbg. The ndbg is for error.